www/src/views/starter/source.js

```diff
@@ -66,6 +67,17 @@ const Source = ({ startersYaml, repoUrl }) => (
       >
         Try this starter
       </span>
+      <a
+        href={`https://codesandbox.io/s/github/${starter.owner}/${starter.stub}`}
+        sx={{
+          "&&": {
+            borderBottom: 0,
+            marginRight: `5px`
```

Comment:
Hey @garethx! 🤗Thanks for taking care of this! 🙏 

May I ask you to use a token from our `space` scale (see https://www.gatsbyjs.org/guidelines/design-tokens/, and https://github.com/gatsbyjs/gatsby/tree/master/www#theme-ui-css-authoring-and-dark-mode for a bit of background info), and replace this line with `marginRight: 2`? This also already slightly increases the value from `5px` to `8px`. I'm not 100% sure is enough whitespace to clearly separate CodeSandbox and Netlify and their icons, so could you take a look at `3` (`12px`), and `4`(`16px`), too? 🙏

Reply:
What do you think to space[4]? <img width="325" alt="Screenshot 2020-01-31 at 22 15 59" src="https://user-images.githubusercontent.com/1830035/73578368-6389e580-4477-11ea-98a2-b604c14246fc.png">

Reply:
[4] looks great!